extensions/escape-game/standalone.mjs

```diff
@@ -200,7 +200,6 @@ class Standalone {
     // Create a new team
     let teamId = randomBytes(32).toString('hex');
     this.teams.set(teamId, new Team(teamId, this.terminalServerGrpcService, this.defaultTerminals));
-    this.teams.get(teamId).startContainers(); // do not wait for the container to start.
```

Comment:
Don't need to start the containers on team creation, since it will be done on showTerminal